odoo/addons/base/models/assetsbundle.py

```diff
@@ -26,6 +26,7 @@
 from odoo.tools import func, misc, transpile_javascript, is_odoo_module, SourceMapGenerator, profiler
 from odoo.tools.misc import file_open, html_escape as escape
 from odoo.tools.pycompat import to_text
+from odoo.release import version as server_version
```

Comment:
@Arcasias This doesn't work for Odoo EE, because `server_version` is imported as a copy of `odoo.release.version` *before* `web_enterprise` is able to append the `+e`.

This can be verified easily in a shell session:

```py
In [1]: odoo.addons.base.models.assetsbundle.server_version
Out[1]: '15.0'

In [2]: odoo.release.version
Out[2]: '15.0+e'
```

So in practice the popup is still triggered every time.

As simple fix is to import `odoo.release` and use `odoo.release.version` in the bus notification directly, or any similar dynamic referencing.

Reply:
Indeed, thanks for noticing!
Here's the fix: https://github.com/odoo/odoo/pull/82687